Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.1-slim
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
```

Comment:
Thank you @frgfm for this PR ! 

Small question: what is the use of the env variable in our case ?

Reply:
It prevents the generation of `.pyc` upon compilation :)

Reply:
Yes, but why should we add this ? only to save disk space ?

Reply:
yes, and simply to keep the number of files to a minimum. It's a very common thing to add in a dockerfile

Reply:
Ok thanks ! 😄